src/olympia/editors/views.py

```diff
@@ -534,18 +534,56 @@ def application_versions_json(request):
     return {'choices': f.version_choices_for_app_id(app_id)}
 
 
+def _get_deleted_versions(addon):
```

Comment:
Could you add a docstring to this function ? I always forget when seeing this code why we need this until I read the code of the class...